base/applications/drwtsn32/main.cpp

```diff
@@ -66,6 +66,10 @@ static void PrintThread(FILE* output, DumpData& data, DWORD tid, ThreadData& thr
                  ctx.Rax, ctx.Rbx, ctx.Rcx, ctx.Rdx, ctx.Rsi, ctx.Rdi);
         xfprintf(output, "r8:%p r9:%p r10:%p r11:%p r12:%p r13:%p r14:%p r15:%p" NEWLINE,
                  ctx.R8, ctx.R9, ctx.R10, ctx.R11, ctx.R12, ctx.R13, ctx.R14, ctx.R15);
+#elif defined(_M_ARM)
+#error ARM architecture
+#elif defined(_M_ARM64)
+#error ARM64 architecture
```

Comment:
No.
Fix the bug or leave it alone, we have too many PR's of you already where you change code with no good reason just to support your aesthetic.
This is both rude, and against our contribution guidelines.

Reply:
Removed.